apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/brevbaker/BrevbakerKlient.kt

```diff
@@ -51,6 +52,22 @@ class BrevbakerKlient(private val client: HttpClient, private val apiUrl: String
     } catch (ex: Exception) {
         throw BrevbakerException("Feil ved kall til brevbaker", ex)
     }
+
+    @OptIn(ExperimentalTime::class)
+    suspend fun genererJSON(brevRequest: BrevbakerRequest): BrevbakerJSONResponse = try {
+        measureTimedValue {
+            client.post("$apiUrl/etterlatte/json") {
+                contentType(ContentType.Application.Json)
+                header("Nav_Call_Id", getXCorrelationId())
```

Comment:
Tror vi har en global konstant for denne

Reply:
Ser faktisk ikkje sånn ut 😮 Men vi burde jo ha det.